packages/tree-select/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This is the "default" `tsconfig`. Becuase it has a default name, editor integrations will pick it up more easily.